source/application/SXRMB/SXRMBAppController.cpp

```diff
@@ -374,7 +374,6 @@ void SXRMBAppController::setupUserInterface()
 	CLSCrossHairGeneratorControlView *crossHairView = new CLSCrossHairGeneratorControlView(SXRMBBeamline::sxrmb()->crossHairGenerator());
 	SXRMBCrystalChangeView *crystalChangeView = new SXRMBCrystalChangeView(SXRMBBeamline::sxrmb()->crystalSelection());
 	CLSJJSlitsView *jjSlitsView = new CLSJJSlitsView(SXRMBBeamline::sxrmb()->jjSlits());
-	jjSlitsView->setDataRange(18, 0);
```

Comment:
I saw you removed the `setDataRange()` function as well. We should test it on SXRMB, whether this will cause an input issue.